monodbassignment/posts.json

```diff
@@ -0,0 +1,21 @@
+{
+  "id":"String datatype", 
+  "author_id": "String datatype(id of author)",
+   "timestamp":"timestamp datatype",
+  "content":{
+    "head":"String datatype",
+    "body":"String datatype",
+    "tags":[],
+    "images":"url"
```

Comment:
you should be array of urls, or if you are taking only one image, its should be called image instead of images.